src/vm/jitinterface.cpp

```diff
@@ -9953,8 +9953,8 @@ void* CEEInfo::getPInvokeUnmanagedTarget(CORINFO_METHOD_HANDLE method,
 
 /*********************************************************************/
     // return address of fixup area for late-bound N/Direct calls.
-void* CEEInfo::getAddressOfPInvokeFixup(CORINFO_METHOD_HANDLE method,
-                                        void **ppIndirection)
```

Comment:
> getAddressOfPInvokeTarget  and  getFunctionEntryPoint 

These two methods return different entrypoints. I do not think they should be merged. (If they were merged, the merged method would need a flag which one of the two different entrypoints is needed ... .)